README.md

```diff
@@ -128,7 +128,7 @@ Some of the statistics you can integrate include:
 - Queries cached
 - Unique clients
 
-Access the API via [`telnet`](https://github.com/pi-hole/FTL), the Web (`admin/api.php`) and Command Line (`pihole -c -j`). You can find out [more details over here](https://discourse.pi-hole.net/t/pi-hole-api/1863).
+Access the API via the Web (`/api/docs`). You can find out [more details over here](https://docs.pi-hole.net/api/).
```

Comment:
In v5 there were 3 different ways to access the web interface: telnet, browser (http) or chronometer.
The `telnet` protocol was accessed using Netcat (`nc`).

Now in v6, there is no telnet, but the `http` protocol can still be accessed using `curl` or a browser (you probably need authentication/password).

Also, there is a new command line option `pihole api <config_option>`.

Not sure how to add show options here... this is a suggestion:

```suggestion
Access the API using:
- your browser: http://pi.hole/api/docs
- `curl`: `curl --connect-timeout 2 -ks "https://pi.hole/api/stats/summary" -H "Accept: application/json"`;
- the command line - examples: `pihole api config/webserver/port` or `pihole api stats/summary`. 
```